hack/update-gofmt.sh

```diff
@@ -42,5 +42,13 @@ find_files() {
     \) -name '*.go'
 }
 
-GOFMT="gofmt -s -w"
-find_files | xargs $GOFMT
+# GOFMT="gofmt -w -s -d -r {}"
+# for file in $(find_files) ; do
+#   echo ${file}
+#   cat ${KUBE_ROOT}/hack/.gofmt | xargs $GOFMT ${file}
+# done
+
+while read rule ; do
+  echo ${rule}
+  find_files | xargs gofmt -w -s -d -r="${rule}"
+done < ${KUBE_ROOT}/hack/.gofmt
```

Comment:
this is origin bash, but because **kubernetes** *.go too long, so will exec error
```
cat .gofmt | xargs -I {} gofmt -w -s -d -r {}  $$(find . -name "*.go" -not -path "./vendor/*" -not -path ".git/*")
```
i write the very slow gofmt. because i am a bash beginner, i really like who can give some help...